code/API_definitions/device-status.yaml

```
@@ -780,52 +855,70 @@ components:
   examples:
     ROAMING_STATUS:
       value:
-        eventSubscriptionId: 123654
-        event:
-          eventType: ROAMING_STATUS
-          eventTime: 2023-01-17T13:18:23.682Z
-          eventDetail:
-            device:
-              phoneNumber: 123456789
-            roaming: true
-            countryCode: 208
-            countryName: FR
+        id: 123654
+        source: supertelco.notificationSendServer12
+        type: org.camaraproject.device-status.v0.roaming-status
+        specversion: "1.0"
+        datacontenttype: application/json
+        data:
+          device:
+            phoneNumber: 123456789
+          roaming: true
+          countryCode: 208
+          countryName: FR
+          subscriptionId: qs15-h556-rt89-1298
+        time: 2023-01-17T13:18:23.682Z
+
     ROAMING_OFF:
       value:
-        eventSubscriptionId: 123655
-        event:
-          eventType: ROAMING_OFF
-          eventTime: 2023-01-17T13:18:23.682Z
-          eventDetail:
-            device:
-              phoneNumber: 123456789
+        id: 123655
+        source: supertelco.notificationSendServer12
+        type: org.camaraproject.device-status.v0.roaming-off
+        specversion: "1.0"
+        datacontenttype: application/json
+        data:
+          device:
+            phoneNumber: 123456788
+          subscriptionId: qs15-h556-rt89-1298
+        time: 2023-01-18T13:18:23.682Z
+
     ROAMING_ON:
       value:
-        eventSubscriptionId: 123655
-        event:
-          eventType: ROAMING_ON
-          eventTime: 2023-01-17T13:18:23.682Z
-          eventDetail:
-            device:
-              phoneNumber: 123456789
+        id: 123656
+        source: supertelco.notificationSendServer12
+        type: org.camaraproject.device-status.v0.roaming-on
+        specversion: "1.0"
+        datacontenttype: application/json
+        data:
+          device:
+            phoneNumber: 123456787
+          subscriptionId: qs15-h556-rt89-1298
+        time: 2023-01-19T13:18:23.682Z
+
     ROAMING_CHANGE_COUNTRY:
       value:
-        eventSubscriptionId: 123655
-        event:
-          eventType: ROAMING_CHANGE_COUNTRY
-          eventTime: 2023-01-17T13:18:23.682Z
-          eventDetail:
-            device:
-              phoneNumber: 123456789
+        id: 123657
+        source: supertelco.notificationSendServer12
```

Changed to _Internet-wide unique URI with a DNS authority_:
```suggestion
        source: https://notificationSendServer12.supertelco.com
```